presto-hudi/src/main/java/com/facebook/presto/hudi/HudiPartitionManager.java

```diff
@@ -78,6 +91,88 @@ public List<String> getEffectivePartitions(
                 partitionPredicate.put(partitionColumn, Domain.all(column.getHiveType().getType(typeManager)));
             }
         }
-        return metastore.getPartitionNamesByFilter(metastoreContext, schemaName, tableName, partitionPredicate);
+
+        // fetch partition names from metastore
```

Comment:
nits: lets remove this kind of obvious comments

Reply:
ack